### PR TITLE
 update API doc based on https://github.com/bosnet/sebak/pull/863/

### PR DESCRIPTION
**Pull request by cmcm2222**
Based on PR#863, API doc is updated. 

please review it.

### CODE REVIEW

**Comment by cmcm2222** [comment]
I didn't add reviewers ^^;;

Would you review this API doc ? @kfangw @soonkuk

**Comment by cmcm2222** [comment]
@kfangw  got it. Thanks for reviewing.

**Comment by kfangw** [comment]
@cmcm2222 Please deploy the DOC when the v0.1.5 released.

File: API_v1/models.md [hunk 1]
```diff
@@ -33,6 +33,13 @@
         + fee: 10000
         + hash: `7nLuyg8radTExzBM2WhG37AwohBwEySBw4vj2xdtdjAs`
         + operation_count: 1
+        + operations:
```
Comment (soonkuk) [inline API_v1/models.md]: In transaction, `block: 241` should be changed
when I call the api in testnet `block` shows block hash

File: API_v1/models.md [hunk 2]
```diff
@@ -33,6 +33,13 @@
...
+            + (array)
+                + H
+                    + type : `payment` - operation type. ex. payment, create-account
+                + B
+                    + target : `GCIKBR6QXYF6VLLTXC7EHKDGAPAOLSRALZTWTLZ2SANBYWMWT6AURQGG`
+                    + amount : `100000` - amount in GON
```
Comment (soonkuk) [inline API_v1/models.md]: `txs, block-height, block-hash , total-txs, total-ops` should be included

Comment (cmcm2222) [reply API_v1/models.md]: when you see the `payment' type. 
https://mainnet.blockchainos.org/api/v1/transactions/EqGR1c3S3dMDD6ZDxusAxuLNJGErb1tzS56Q2QNYUUoW

you can see that it is different with inflation or collect-tx-fee.

File: API_v1/models.md (hunk 1 above)
Comment (cmcm2222) [reply API_v1/models.md]: thank :)
👍

File: API_v1/models.md (hunk 2 above)
Comment (soonkuk) [reply API_v1/models.md]: Ok I got it.

**Pull request merged**